testutil/ciao-down/ciao_down.go

```diff
@@ -494,6 +495,9 @@ func connect(ctx context.Context, errCh chan error) {
 			case <-ctx.Done():
 				errCh <- fmt.Errorf("Cancelled")
 				return
+			case <-timeout.C:
+				fmt.Println("\nCannot connect to VM, please make sure it has been started:\n\t$ ciao-down start")
+				break DONE
```

Comment:
This will still try to ssh into the instance which is unlikely to succeed.  We should probably just return here sending an error down the errCh.

Reply:
AH, I meant to return at this point... you're right it will still try to ssh in, and if you put the timeout low enough it can hit the case where it prints out "Cannot connect" and then ssh's in successfully.